shopfloor_base/models/shopfloor_scenario.py

```diff
@@ -77,4 +77,4 @@ def _normalize_key(name):
         return slugify(name).replace("-", "_")
 
     def has_option(self, key):
-        return self.options.get(key, False)
+        return key in self.options.keys()
```

Comment:
Don't change this, set new option to True

Reply:
ok